.rubocop_common.yml

```diff
@@ -0,0 +1,63 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-minitest
```

Comment:
If we really want this, I'd suggest to actually split core cops, performance, rails and minitest cops alltogether. Therefore 4 include YAMLs would be provided for optional inclusion.